examples/controls_example/public/react_controls/initialize_default_control_api.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+
+import { ControlWidth } from '@kbn/controls-plugin/common';
+import { SerializedPanelState } from '@kbn/presentation-containers';
+import { StateComparators } from '@kbn/presentation-publishing';
+
+import {
+  ControlApiInitialization,
+  ControlStateManager,
+  DefaultControlApi,
+  DefaultControlState,
+} from './types';
+
+export type ControlApi = ControlApiInitialization<DefaultControlApi>;
+
+export const initializeDefaultControlApi = (
```

Comment:
No action item required here at the moment, but I'm starting to wonder if the overhead of having `defaultControl` and `dataControl` as two separate concerns is overkill at the moment because for the near future we are only planning on having data controls.

Reply:
Time slider isn't technically a `dataControl` - it is the only one that **will** use `defaultControl` but **won't** use `dataControl` (because it doesn't require a data view). So I separated them out with that in mind 😅

Reply:
Ah yes I remember this discussion, my bad!